core/SDL.carp

```diff
@@ -56,6 +59,12 @@
     (register init (Fn [] SDL_Event))
     (register type (Fn [(Ref SDL_Event)] SDL_EventType))
     (register keycode (Fn [(Ref SDL_Event)] SDL_Keycode))
+    ;(register window (Fn [(Ref SDL_Event)] SDL_WindowEvent))
```

Comment:
Should the commented version stay in there?

Reply:
Yes, because the intent is to not rely on deftemplate in the future. I'll add a note.